### PR TITLE
Introduce Spark console view message bars

### PR DESCRIPTION
**Pull request by wezhang**
And added Spark batch Arcadia history web into message bar.

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkJobLogConsoleView.java [hunk 1]
```diff
@@ -45,7 +44,7 @@ public SparkJobLogConsoleView(@NotNull Project project) {
         super(project, true);
 
         // set `usePredefinedMessageFilter = false` to disable predefined filter by console view and avoid filter conflict
```
Comment (lcadzy) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkJobLogConsoleView.java]: >  // set `us [](start = 6, length = 12)

seems need to remove this line to ConsoleViewWithMessageBars.kt

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ArcadiaSparkBatchJob.java [hunk 2]
```diff
@@ -78,6 +79,10 @@ public ArcadiaSparkBatchJob(final @NotNull SparkSubmissionParameter submissionPa
     @Override
     public Observable<SimpleImmutableEntry<MessageInfoType, String>> getSubmissionLog() {
         // No batches/{id}/log API support yet
+        getCtrlSubject().onNext(new SimpleImmutableEntry<>(
+                MessageInfoType.HtmlPersistentMessage,
```
Comment (lcadzy) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ArcadiaSparkBatchJob.java]: can we reuse the ctrlInfo method ?

**Comment by lcadzy** [comment]
Thank you, Wei!

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ArcadiaSparkBatchJob.java (hunk 2 above)
Comment (wezhang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/ArcadiaSparkBatchJob.java]: It's a different behavior and presentation.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkJobLogConsoleView.java (hunk 1 above)
Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkJobLogConsoleView.java]: Clean up it later. 😄

**Pull request merged**